ote/src/clj/ote/services/transport.clj

```diff
@@ -541,9 +540,90 @@
   [db operator-id]
   (http/transit-response (fetch-operator-users db {:operator-id operator-id})))
 
+(defn add-user-to-operator [email db new-member requester operator]
```

Comment:
Implementation relies that `create-member!` won't fail. 
Is that ok? Should a failing insert throw an exception or return an error response?

Reply:
Really good point. I get into that. But it is very unlikely that simple insert will fail.

Reply:
Same logic is used earlier so I think that this is ok for now.

Reply:
Ok. An easy way out would be to throw an exception if nil member is returned ==> http request completed automagically with status 500 or similar.